modules/react/button/lib/PrimaryButtonKing.tsx

```diff
@@ -0,0 +1,423 @@
+      },
+    };
+
+    const activeStyles = {
+      '&:active, &:focus:active, &:hover:active': {
+        backgroundColor: colors.active.background,
+        borderColor: colors.active.border,
+        color: colors.active.label,
+        ...(colors.active.labelData && {
+          ['.' + buttonLabelDataClassName]: {
+            color: colors.active.labelData,
+          },
+        }),
+        ...(colors.active.icon && getIconColorSelectorsNew(theme, colors.active.icon, false)),
+      },
+    };
+
+    return {
+      ...baseStyles,
+      '&:focus': {
+        backgroundColor: colors.focus.background,
+        borderColor: colors.focus.border,
+        color: colors.focus.label,
+        ...(colors.focus.focusRing || focusRing({separation: 2}, theme)),
+        ...(colors.focus.labelData && {
+          ['.' + buttonLabelDataClassName]: {
+            color: colors.focus.labelData,
+          },
+        }),
+        ...(colors.focus.icon && getIconColorSelectorsNew(theme, colors.focus.icon, false)),
+      },
+
+      ...activeStyles,
+      ...hoverStyles,
+      '&:disabled, &:active:disabled, &:focus:disabled, &:hover:disabled': {
+        backgroundColor: colors.disabled.background,
+        borderColor: colors.disabled.border,
+        color: colors.disabled.label,
+        ...(colors.disabled.icon && getIconColorSelectorsNew(theme, colors.disabled.icon, false)),
+        ...(colors.disabled.labelData && {
+          ['.' + buttonLabelDataClassName]: {
+            color: colors.disabled.labelData,
+          },
+        }),
+      },
+      ...mouseFocusBehavior({
+        '&:focus': {
+          ...baseStyles,
+          outline: 'none',
+          boxShadow: 'none',
+          animation: 'none',
+          ...hoverStyles,
+          ...activeStyles,
+        },
+      }),
+    };
+  }
+);
+
+const getMinWidthStyles = (children: React.ReactNode, size: ButtonSizes) => {
+  switch (size) {
+    case 'large':
+      return children ? '112px' : '48px';
+    case 'medium':
+      return children ? '96px' : '40px';
+    case 'small':
+      return children ? '80px' : '32px';
+    case 'extraSmall':
+      return children ? 'auto' : '24px';
+    default:
+      return children ? '96px' : '40px';
+  }
+};
+
+const getPaddingStyles = (children: React.ReactNode, size: ButtonSizes) => {
+  switch (size) {
+    case 'large':
+      return children ? `0 ${space.l}` : 0;
+
+    case 'medium':
+      return children ? `0 ${space.m}` : 0;
+    case 'small':
+      return children ? `0 ${space.s}` : 0;
+    case 'extraSmall':
+      return children ? `0 ${space.xs}` : 0;
+    default:
+      return children ? `0 ${space.m}` : 0;
+  }
+};
+
+export const PrimaryButtonKing = createComponent('button')({
+  displayName: 'PrimaryButton',
+  Component: (
+    {
+      // TODO: Fix useTheme and make it a real hook
+      // eslint-disable-next-line react-hooks/rules-of-hooks
+      theme = useTheme(),
```

Comment:
I don't see `theme` used inside the render function...

If it was, I'd like this to be fixed - it is a hook/not-hook. Either it is a hook or it isn't.

Proper use would be something like:

```jsx

const theme = useTheme()
```

Real hook. Not from props. Drop `Themeable` if it isn't being used. I think React Context + hooks makes more sense than whatever is going on here